<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>749</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777194</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agenda telefóbica</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/logo.ico</normaloff>img/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="panPrincipal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>1001</width>
      <height>681</height>
     </rect>
    </property>
    <widget class="QWidget" name="pesContactos">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>10</y>
        <width>821</width>
        <height>146</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QLabel" name="lblfechaalta">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Fecha alta:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QTextEdit" name="txtciudad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lbllnotas">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Notas:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="txtnombre">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblmovil">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Móvil:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="txtmovil">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>24</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QTextEdit" name="txtfechaalta">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QTextEdit" name="txtnotas">
         <property name="minimumSize">
          <size>
           <width>151</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="txtemail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="txtid">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblnombre">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblemail">
         <property name="minimumSize">
          <size>
           <width>47</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblid">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>id:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblciudad">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Ciudad:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>821</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="botonalta">
           <property name="text">
            <string>ALTA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botonmodificar">
           <property name="text">
            <string>MODIFICAR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botoneliminar">
           <property name="text">
            <string>ELIMINAR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Email</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Movil</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ciudad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Notas</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Alta</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkhistorico">
         <property name="text">
          <string>Histórico</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuGesti_n">
    <property name="title">
     <string>Gestión</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionCrear_Backup"/>
    <addaction name="actionRestaurar_Backup"/>
    <addaction name="separator"/>
    <addaction name="actionExportar_Datos"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuGesti_n"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCrear_Backup">
   <property name="text">
    <string>Crear Backup</string>
   </property>
  </action>
  <action name="actionRestaurar_Backup">
   <property name="text">
    <string>Restaurar Backup</string>
   </property>
  </action>
  <action name="actionExportar_Datos">
   <property name="text">
    <string>Exportar Datos</string>
   </property>
  </action>
  <action name="action_limpiar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/limpiar.ico</normaloff>../img/limpiar.ico</iconset>
   </property>
   <property name="text">
    <string>boton limpiar</string>
   </property>
   <property name="toolTip">
    <string>Limpiar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
